src/shared/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Redirect,
+  Route,
+  Switch,
+  BrowserRouter as Router,
+} from "react-router-dom";
+import { ThemeProvider } from "@material-ui/core/styles";
+
+import { AuthProvider } from "../context/auth";
+import Login from "../pages/Login";
+import ProjectREAD from "./ProjectREAD";
+import PrivateRoute from "./PrivateRoute";
+import theme from "../theme";
+
+function App() {
+  return (
+    <ThemeProvider theme={theme}>
+      <AuthProvider>
+        <Router>
+          <Switch>
+            <Route exact path="/login" component={Login} />
+            <PrivateRoute
+              exact
+              path={["/", "/sessions"]}
```

Comment:
i don't love listing the paths here bc it's a bit repetitive of what's in the ProjectREAD component - lmk if there's a better way to do this!

Reply:
Just a little confused, why do we need to list the paths in both places?